persistence/sql/migrations/20200527215731_client.up.sql

```diff
@@ -0,0 +1 @@
+ ALTER TABLE hydra_client ADD token_endpoint_auth_signing_alg VARCHAR(10) NOT NULL DEFAULT '';
```

Comment:
Nice! This would also need a down migration, plus a test migration ([example](https://github.com/ory/hydra/blob/c92ab20dae186585f1cd3f9388791d3ae9d24ebb/persistence/sql/migratest/testdata/20190100000006_testdata.sql)) :)

This could also `DEAFULT` to `RS256`